### PR TITLE
[CNTOOLS] - Initial commit for a CNTools offline mode

### PR DESCRIPTION
**Pull request by Scitz0**
TODO (before merge):
- Rename gLiveView.sh URL_RAW branch back to master
- Rename BRANCH value back to master in prereqs.sh

Adds -o command line parameter to run CNTools in offline mode. 
In addition to offline mode, several additional improvements and modifications have been made.
See the individual commit logs for details.

### ToDo - Offline functionality
- [x] Simplified offline Show|List views of wallets and pools
- [x] Wallet delete without balance check option
- [x] Pool KES rotation
- [x] Allow creation of staging tx files using ttl as input in an online/offline hybrid mode.
  - [x] To transfer funds
  - [x] Withdraw Rewards
  - [x] Delegate
  - [x] Register pool
  - [x] Modify pool
  - [x] Retire pool
- [x] Add tx signing (to sign built tx with keys on offline computer in offline mode)
- [x] Add tx submission (to submit signed tx file exported from the offline computer)
- [x] Check `pool >> show` stake distribution, always 0.
- [x] Add a way to transfer all files except important signing keys to hot node, ie maybe a modified backup
- [x] Mnemonic import integration according to guide at https://gist.github.com/ilap/3fd57e39520c90f084d25b0ef2b96894
- more? (see individual commit logs)

### CODE REVIEW

**Comment by rdlrt** [comment]
Waiting for @redoracle to test the docker flow once

File: scripts/cnode-helper-scripts/cntools.sh [hunk 1]
```diff
@@ -49,7 +49,7 @@ shift $((OPTIND -1))
 # get common env variables
 if ! . "${CNODE_HOME}"/scripts/env; then
   [[ ${CNTOOLS_MODE} = "CONNECTED" ]] && exit 1
-  echo -e "\n${FG_RED}ERROR${NC}: CNTools run in offline mode and failed to automatically grab common env variables\nPlease uncomment all variables in 'User Variables' section and set values manually\n"
```
Comment (rdlrt) [inline scripts/cnode-helper-scripts/cntools.sh]: 👍

**Comment by redoracle** [comment]
> Waiting for @redoracle to test the docker flow once

I will come back to you within today 🙏

**Comment by rdlrt** [comment]
Moved to #518  to release in alpha mode before its merged to master

**Pull request closed without merging**